rse-bash.Rmd

```diff
@@ -0,0 +1,3902 @@
+# Bash Shell {#bash}
+
+## Questions {#bash-questions}
+
+```{r, child="questions/bash.md"}
+```
+
+## Objectives {#bash-objectives}
+
+```{r, child="objectives/bash.md"}
+```
+
+## Introduction {#bash-intro}
+
+At a high level, computers do four things:
+
+-   run programs
+-   store data
+-   communicate with each other, and
+-   interact with us
+
+They can do the last of these in many different ways,
+including through a keyboard and mouse, touch screen interfaces, or using speech recognition systems.
+While touch and voice interfaces are becoming more commonplace, most interaction is still
+done using traditional screens, mice, touchpads and keyboards.
+
+The **graphical user interface** (GUI) is the most widely used way to interact with
```

Comment:
Please use `[graphical user interface](glossary.html#gui)` for all glossary references. (The key `gui` after the `#` should match something in `gloss.md`.)